packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -0,0 +1,6 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+
+export const styles = css`
+    ${display('flex')}
```

Comment:
Good work on bringing up the skeleton! 

Planning to import theme based design tokens for fonts & their tests into this PR?

Reply:
A reference on how we set it in text field component [here](https://github.com/ni/nimble/blob/cb2bf64825e963d86ebe8bd740ebc52196edcd76/packages/nimble-components/src/text-field/styles.ts#L30-L39)

Reply:
I don't think it's strictly needed right now since the entire contents are placeholder and will be replaced with something official in a future PR. But if you feel like it, I can't see any harm in using the Nimble body font and text color for now.

Reply:
@vikisekarNI, @jattasNI, I have updated the font color using the nimble body font and color as suggested in this PR. Anyway, we will be adding these styles in the next PR for the viewer component, so added them here. Please review and approve the chromatic test as you see fit.

Reply:
Approved!